kedro-datasets/CONTRIBUTING.md

```diff
@@ -23,12 +23,48 @@ If you have already checked the [existing issues](https://github.com/kedro-org/k
 
 If you have new ideas for Kedro-Datasets then please open a [GitHub issue](https://github.com/kedro-org/kedro-plugins/issues) with the label `enhancement`. Please describe in your own words the feature you would like to see, why you need it, and how it should work.
 
-### Contribute a new dataset
+## Contribute a new dataset
 
 If you're unsure where to begin contributing to Kedro-Datasets, please start by looking through the `good first issue` and `help wanted` on [GitHub](https://github.com/kedro-org/kedro-plugins/issues).
 If you want to contribute a new dataset, read the [tutorial to create and contribute a custom dataset](https://docs.kedro.org/en/stable/data/how_to_create_a_custom_dataset.html) in the Kedro documentation.
 Make sure to add the new dataset to `kedro_datasets.rst` so that it shows up in the API documentation and to `static/jsonschema/kedro-catalog-X.json` for IDE validation.
 
+Below is a guide to help you understand the process of contributing a new dataset, whether it falls under the category of regular or experimental datasets.
+
+### Difference between regular and experimental datasets
+
+#### Regular datasets
```

Comment:
```suggestion
#### 1st party supported datasets
```

I think regular is a bit too vague, something like this is really explicit from the title

Reply:
I've just changed this to "Core" after @deepyaman's suggestion. Does that sound better?

Reply:
Yeah much better!